metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures mcrouter'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '3.0.0'
+version          '3.0.1'
```

Comment:
This feels like a minor version bump, not a patch.

Reply:
If you don't feel like bumping the minor version, I can go ahead and merge, just let me know.

Reply:
I bumped the version to `3.1.0`